scripts/multiply_by_x.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
```

Comment:
Add a comment to this and the other test scripts that they are for testing purposes.